### PR TITLE
fix: Edit links in Newcomers_Guide.rst

### PR DESCRIPTION
**Pull request by adamki**
### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog).
- [x] I have followed the [guidelines for the commit body](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#issue-reference).
- [x] I have [run all the tests](http://coala.readthedocs.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### Reviewers

Fixes https://github.com/coala/coala/issues/3239

### CODE REVIEW

**Comment by Udayan12167** [comment]
Still not following the commit message guidelines. The shortlog should read like `Filename.rst: Change performed`

**Comment by adamki** [comment]
@dracarys09 Initially, i started writing a commit body, however, per instructions [here](https://github.com/coala/coala/blob/master/docs/Developers/Writing_Good_Commits.rst#commit-body), i opted to leave it out based on the complexity (or lack of) of this particular change.

**Comment by aptrishu** [comment]
Point the links to docs. @adamki 
https://docs.coala/io

**Comment by adamki** [comment]
@aptrishu  with all due respect, the link you shared does not have a valid `https` and the content is not loading either. 

Also, i was instructed to link to the repo, not the docs.

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -209,8 +208,7 @@ you can do is wait. The best thing you can do while at this step is review
 other people's PRs. Not only will this help the maintainers with the workload,
 but this is one of the three core steps towards becoming a full-norm coalaian.
 
-For more information about reviewing code, check out
-:doc:`this link.<Review>`
+For more information about reviewing code, check out this `link <https://github.com/coala/coala/blob/master/docs/Developers/Review.rst>`_.
```
Comment (Mariatta) [inline docs/Developers/Newcomers_Guide.rst]: This should be http://coala.io/reviewing

**Comment by Mariatta** [comment]
@adamki Sorry about this, there has been some confusions among the developers of what the fix should be for this issue.
What we really wanted is to update those links to the short urls like coala.io/git  coala.io/commit and coala.io/reviewing

Thanks.

**Comment by adamki** [comment]
@aptrishu @Mariatta np. thanks for clarification

**Comment by Mariatta** [comment]
ack b9d3b4f

**Comment by ashwini0529** [comment]
We can elaborate the commit message. That looks unclear about `Edit links`.

**Comment by Mariatta** [comment]
unack b9d3b4f

**Comment by sils** [comment]
possible overlap with https://github.com/coala/coala/pull/3220/files ! Maybe conflicting.

**Comment by Mariatta** [comment]
Please rebase. Thanks.

**Comment by Mixih** [comment]
Dont close!

**Comment by Mixih** [comment]
Uh-oh! You have have the dreaded merge commit. This is probably because you pushed the rebase(update branch from master) button in github. This is not the proper way of doing this and will break many things. Please do an interactive rebase on the commit before the merge commit and remove the merge commit line(see http://stackoverflow.com/questions/17577409/git-remove-merge-commit-from-history). See https://api.coala.io/en/latest/Developers/Git_Basics.html#rebasing for the proper way to rebase.
(it's okay, we all make mistakes in git ;) )

**Comment by ashwini0529** [comment]
You have a merge commit :/

**Comment by sims1253** [comment]
marking this wip as it needs a rebase

**Comment by Mixih** [comment]
@adamki Can you come back and finish off this issue? There is only very little work left and it would close one of out pending pulls. Thanks!

**Comment by adamki** [comment]
@Mixih done

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -108,8 +108,7 @@ We use GitHub to manage our repository. If you're not familiar with git, we
 strongly recommend following a tutorial, such as `this one <https://try.github.io/levels/1/challenges/1>`_.
 
 We also have a page dedicated to git commands that will help you learn the
-basics,
-:doc:`here <Git_Basics>`.
+basics, `here <http://coala.io/git>`_.
```
Comment (Mixih) [inline docs/Developers/Newcomers_Guide.rst]: , -> : (better grammatically)

File: docs/Developers/Newcomers_Guide.rst [hunk 3]
```diff
@@ -140,8 +139,8 @@ Now you can start working on it.
 
 .. note::
 
-    Before starting to write your first commit, check out this link:
-    :doc:`Writing good commits <Writing_Good_Commits>`.
+    Before starting to write your first commit, check out this
+    link, `Writing good commits <http://coala.io/commit>`_.
```
Comment (Mixih) [inline docs/Developers/Newcomers_Guide.rst]: , -> : (better grammatically)

File: docs/Developers/Newcomers_Guide.rst (hunk 2 above)
Comment (adamki) [inline docs/Developers/Newcomers_Guide.rst]: you want me to add a `:`?

**Comment by Mixih** [comment]
ack 6f11eea

**Comment by Mixih** [comment]
@rultor merge

**Pull request merged**